slam-libraries/viam-cartographer/scripts/run_cartographer.sh

```diff
@@ -12,56 +12,30 @@ else
 fi
 
 # ---- Edit based on your needs:
-DATE="June01"
-
-MAPPING="true"
-LOCALIZATION="false"
-UPDATE="false"
+DATE="October01"
 
 DATA_BASE_DIRECTORY="$HOME/viam/lidar_data"
-MAPPING_DATA_DIRECTORY="$DATA_BASE_DIRECTORY/data_Feb_11_2022_small"
+MAPPING_DATA_DIRECTORY="$DATA_BASE_DIRECTORY/data_Feb_11_2022_small/data"
 LOCALIZATION_DATA_DIRECTORY="$DATA_BASE_DIRECTORY/data_Feb_24_2022_printer_room"
 UPDATE_DATA_DIRECTORY="$DATA_BASE_DIRECTORY/data_Feb_24_2022_printer_room"
 
 DESCRIPTION="_refactor"
 OUTPUT_DIRECTORY="pics${DESCRIPTION}_${DATE}"
 MAP_OUTPUT_NAME="map${DESCRIPTION}_${DATE}.pbstream"
 
-PICTURE_PRINT_INTERVAL="50"
-
-MAPPING_STARTING_SCAN_NUMBER="0"
-LOCALIZATION_STARTING_SCAN_NUMBER="0"
-UPDATE_STARTING_SCAN_NUMBER="0"
 # ----
 
-CONFIGURATION_DIRECTORY="../lua_files"
-CONFIGURATION_MAPPING_BASENAME="mapping_new_map.lua"
```

Comment:
what did we want to do with the `.lua` files? I remember we mentioned maybe treating them like `ORBVoc.txt` but idk if we ever did anything with that in the specs

Reply:
I don't remember tbh - probably something that'll be good to discuss in our next (or next->next) SLAM meeting

Reply:
Adding an item to the agenda now